Sources/WhisperKit/Core/AudioStreamTranscriber.swift

```diff
@@ -194,11 +206,11 @@ public actor AudioStreamTranscriber {
         }
     }
 
-    private func transcribeAudioSamples(_ samples: [Float]) async throws -> TranscriptionResult? {
+    private func transcribeAudioSamples(_ samples: [Float]) async throws -> TranscriptionResult {
```

Comment:
this is a private function so no risk of a breaking change here. Since it's a throwing function which returns the result of `TranscribeTask.run` which is non optional I'd leave as non optional